js/main.js

```diff
@@ -102,96 +111,110 @@ renderPins();
 
 // создаем DOM-элемент объявления (карточка объявления)
 
-var cardTemplate = document.querySelector('#card').content.querySelector('article');
-
-// создаем тип жилья
-var createTypeCard = function (card) {
-  switch (true) {
-    case card.offer.type === 'flat':
-      card.offer.type = 'Квартира';
-      break;
-
-    case card.offer.type === 'bungalo':
-      card.offer.type = 'Бунгало';
-      break;
-
-    case card.offer.type === 'house':
-      card.offer.type = 'Дом';
-      break;
-    case card.offer.type === 'palace':
-      card.offer.type = 'Дворец';
-      break;
-  }
-  return card.offer.type;
-};
-
-// создаем удобства в месте проживания
-var createFeature = function (container, feature) {
-  for (var i = 0; i < feature.length; i++) {
+// создаем фрагмент удобств
+var createFeatureFragment = function (adData) {
+  var featureFragment = document.createDocumentFragment();
+  for (var i = 0; i < adData.offer.features.length; i++) {
     var featureItem = document.createElement('li');
     featureItem.classList.add('popup__feature');
-    featureItem.classList.add('popup__feature--' + feature[i]);
-    container.querySelector('.popup__features').appendChild(featureItem);
+    featureItem.classList.add('popup__feature--' + adData.offer.features[i]);
```

Comment:
можно добавить сразу несколько классов
```suggestion
    featureItem.classList.add('popup__feature', 'popup__feature--' + adData.offer.features[i]);
```